proto/mezo/bridge/v1/query.proto

```diff
@@ -22,4 +35,47 @@ message QueryParamsRequest {}
 message QueryParamsResponse {
   // params holds all the parameters of this module.
   Params params = 1 [ (gogoproto.nullable) = false ];
-}
\ No newline at end of file
+}
+
+// QueryAssetsUnlockedSequenceTipRequest is request type for the
+// Query/AssetsUnlockedSequenceTip RPC method.
+message QueryAssetsUnlockedSequenceTipRequest {}
+
+// QueryAssetsUnlockedSequenceTipResponse is response type for the
+// Query/AssetsUnlockedSequenceTip RPC method.
+message QueryAssetsUnlockedSequenceTipResponse {
+  // sequence_tip is the current assets unlocked sequence tip.
+  string sequence_tip = 1 [
+    (gogoproto.customtype) = "cosmossdk.io/math.Int",
+    (gogoproto.nullable) = false
+  ];
+}
+
+// QueryAssetsUnlockedEventsRequest is request type for the
+// Query/AssetsUnlockedEvents RPC method.
+message QueryAssetsUnlockedEventsRequest {
+  // sequence_start is the start of the sequence range (inclusive). If the
+  // underlying pointer is set to nil, the range is unbounded on the lower side.
+  // Notice that it is the underlying pointer that can be set to nil, not the
+  // sequence_start itself.
```

Comment:
Should we make sequence_start and _end optional to reflect what's said in the comment?

Reply:
@tomaszslabon indeed you can do as you say. It's just that making it optional would indicate that they are in fact optional via the protobuf type system, making those comment obsolete pretty much, and so easier to maintain (IMO) but I'm not picky about it to be fair.